template.yaml

```diff
@@ -150,7 +158,7 @@ Resources:
           BUILDKITE_QUEUE:               !Ref BuildkiteQueue
           AGENTS_PER_INSTANCE:           !Ref AgentsPerInstance
           CLOUDWATCH_METRICS:            "1"
-          DISABLE_SCALE_IN:              "1"
+          DISABLE_SCALE_IN:              !Ref DisableScaleIn
```

Comment:
While this changes the value passed from `"1"` to `"true"` the lambda code already [supports](https://github.com/buildkite/buildkite-agent-scaler/blob/8f7d8e2caca3808a75a33164a0720a5dae479286/lambda/main.go#L176) this.